cmd/tracee-ebpf/capabilities.go

```diff
@@ -0,0 +1,110 @@
+package main
+
+import (
+	"fmt"
+	"github.com/aquasecurity/tracee/pkg/capabilities"
+	tracee "github.com/aquasecurity/tracee/pkg/ebpf"
+	"github.com/syndtr/gocapability/capability"
+)
+
+// IKernelVersionInfo is an interface to check kernel version
+type IKernelVersionInfo interface {
+	// CompareOSBaseKernelRelease compare given kernel version to current one.
+	// The return value is -1, 0 or 1 if given version is less,
+	// equal or bigger, respectively, than running one.
+	CompareOSBaseKernelRelease(string) int
+}
```

Comment:
I don't think this should be in this file.
If it is required for the tests (mockOSinfo), maybe put it there?

Reply:
But I have to use it in this file as the type received in the functions.
I don't think that a code that is used by the program should be in a test file.